documentation/staging/content/release-notes.md

```diff
@@ -8,55 +8,110 @@ draft: false
+* The operator added a [schema conversion webhook]({{< relref "/managing-operators/conversion-webhook.md" >}}) that automates the upgrade of `weblogic.oracle/v8` Domain resources to `weblogic.oracle/v9`. Customers do not need to take any action to begin using the latest Domain schema; however, the team recommends starting with "v9" Domain resources for any new projects. The operator provides a library that can be used to update existing "v8" Domain YAML files to "v9".
+* This release also introduces a new custom resource, the `weblogic.oracle/v1` Cluster resource. This Cluster resource configures a specific WebLogic cluster and allows customers to scale that cluster using the Kubernetes Horizontal Pod Autoscaling (HPA) or similar technologies. The Cluster resource schema is similar to the former `weblogic.oracle/v8` Domain resource `.spec.clusters[*]` content.
+* Customers who install a single WebLogic Kubernetes Operator per Kubernetes cluster (most common use case) can upgrade directly from any 3.x operator release to 4.0.0. The Helm chart for 4.0.0 will automatically install the schema conversion webhook.
+* For those customers who install more than one WebLogic Kubernetes Operator in a single Kubernetes cluster, you must upgrade every operator to at least version 3.4.1 before upgrading any operator to 4.0.0. As the 4.0.0 Helm chart now also installs a schema conversion webhook, you will want to use the Helm chart option to install this webhook in its own namespace prior to installing any of the 4.0.0 operators.
+* Several [Helm chart default values have been changed]({{< ref "#changes-to-helm-chart" >}}). Customers who upgrade their 3.x installations using the `--reuse-values` option during the Helm upgrade will continue to use the values from their original installation.
+
+##### Changes to domain schema
+
+* Added `.spec.configuration.model.auxiliaryImages` for simplified configuration of auxiliary images.
+* Added several additional advanced settings related to auxiliary images including `.spec.configuration.model.auxiliaryImageVolumeMountPath`, `.spec.configuration.model.auxiliaryImageVolumeMedium`, and `.spec.configuration.model.auxiliaryImageVolumeSizeLimit`.
+* Removed `.spec.serverPod.auxiliaryImages` and `.spec.auxiliaryImageVolumes` as part of the simplification effort.
+* Content under `.spec.clusters[*]` is moved to the new Cluster resource `.spec` and is replaced in the Domain schema with references to the Cluster resources.
+* Removed `allowReplicasBelowMinDynClusterSize` field from `.spec.` and `.spec.clusters[*]` as cluster minimums must now be configured through autoscaling.
+* Removed `.spec.configuration.istio` as this section is no longer required.
+* The previously deprecated fields `.spec.domainHomeInImage`, `.spec.configOverrides`, and `.spec.configOverrideSecrets` have been removed.
```

Comment:
Consider adding something like: use `.spec.domainHomeSourceType`, `.spec.configuration.overridesConfigMap`, and `.spec.configuration.secrets` instead.